src/wolfgame.py

```diff
@@ -263,7 +259,7 @@ def on_banlist_end(cli, svr, nick, chan, *etc):
     else:
         cli.who(botconfig.CHANNEL, "%uhsnfa")
 
-@hook("mode")
+@hook("mode") # not bothering to update this function - it was already re-implemented in channels.Channel.update_modes
```

Comment:
if this function is obsolete by the refactor, mark with a TODO or FIXME comment indicating it should be removed at a later date. Otherwise, remove the comment entirely; code comments are for explaining why the code is doing what it is doing or how to do certain things, not for general remarks about a process you are performing. Reminders for later using a tag like TODO or FIXME allows us to easily scan the repo for such tags to ensure that they eventually get done.